resources/istio/files/istio-operator-cluster-evaluation.yaml

```diff
@@ -82,7 +82,10 @@ spec:
           requests:
             cpu: 25m
             memory: 32Mi
+        serviceAnnotations:
+          "service.beta.kubernetes.io/aws-load-balancer-type": "nlb"
```

Comment:
This is strictly AWS-related setting. It should not be hardcoded in a file common to all environments. The proper way to do it:
- define aws-related settings somewhere in values.yaml
- in the aws-setting section prepare "enabled" flag with default value = false!
- if enabled embed the settings from values.yaml here.

Reply:
Thanks